ambry-store/src/main/java/com/github/ambry/store/BlobStoreStats.java

```diff
@@ -175,6 +185,13 @@ static StatsSnapshot convertStoreUsageToContainerStatsSnapshot(Map<Short, Map<Sh
     return new Pair<>(logSegmentValidSizeResult.getFirst(), totalValidSize);
   }
 
+  /**
+   * Get valid data size asynchronously.
+   */
+  Pair<Long, Long> getValidSizeAsync() {
```

Comment:
This is not async. Name it to `getCachedValidSize`?

Reply:
Renamed.